docker/jukebox.Dockerfile

```diff
@@ -46,6 +47,7 @@ RUN [ "$(uname -m)" = "aarch64" ] && ARCH="arm64" || ARCH="$(uname -m)"; \
 
 RUN export ZMQ_PREFIX=${PREFIX} && export ZMQ_DRAFT_API=1
 RUN pip install -v --no-binary pyzmq --pre pyzmq
```

Comment:
I think this is a (the) bug. The exports have to be in the same RUN statement

Reply:
Good catch. Thanks!

Reply:
fixed this